examples/jsm/objects/BatchedMesh.js

```diff
@@ -563,17 +563,29 @@ class BatchedMesh extends Mesh {
 
 	}
 
-	copy() {
+	copy( source ) {
```

Comment:
Can we have `copy` return `this` to make it consistent with parent classes `Object3D` and `Mesh`?